### PR TITLE
Add RPM spec boiler plate code

### PR DESCRIPTION
**Pull request by mandemadhura**

### CODE REVIEW

File: dodo/dodo.spec [hunk 1]
```diff
@@ -0,0 +1,31 @@
+Name:   dodo
+Version:    1.1
+Release:	1%{?dist}
+Summary:	RPM Package for dodo test runner
+
+License:    test_runner
```
Comment (vbmade2000) [inline dodo/dodo.spec]: Please change license to None for now. We need to decide what license to use.

File: dodo/dodo.spec [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+Source0:    %{name}-%{version}.tgz
+BuildRoot:  %(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX)
+
+%description
+Test Runner RPM Package
```
Comment (vbmade2000) [inline dodo/dodo.spec]: Functional test runner for Python based application.

**Review by vbmade2000 — CHANGES_REQUESTED** [review]
Few comments.

File: dodo/dodo.spec (hunk 1 above)
Comment (mandemadhura) [reply dodo/dodo.spec]: Okay.

File: dodo/dodo.spec (hunk 2 above)
Comment (mandemadhura) [reply dodo/dodo.spec]: Okay.

**Pull request merged**